CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
 Fix: Fixed a bug about pagination problem due to missing ErrorCode with details in the case of queries with forwarding to context providers.  (Issue #945)
+Add: New convop "GET /v2/entities" (Issue #947)
```

Comment:
It was my understanding that in v2 we are no longer using the term convop, aren't we?